packaging/ansible-runner-service-project/project/roles/ovirt-host-upgrade/tasks/main.yml

```diff
@@ -41,6 +41,10 @@
   set_fact:
     previous_vdsm_package: "{{ ansible_facts.packages['vdsm'] | first }}"
 
+- name: Clean yum cache
+  command:
+    cmd: yum clean packages
+
```

Comment:
Do we want to abort upgrade if `yum clean packages` returns non-zero status? Or wouldn't it be better to continue and hope for the best?